tests/gcp/test-utils_gcp.R

```diff
@@ -0,0 +1,195 @@
+tar_test("gcp_gcs_exists()", {
+  skip_if_no_gcp()
+  auth_gcp()
+  bucket <- random_bucket_name()
+  # needs to be a GCP project the tester auth has access to
+  projectId <- "gcp-setup-demo" 
```

Comment:
Would you change `projectId <- "gcp-setup-demo"` to `project <- Sys.getenv("GCE_DEFAULT_PROJECT_ID")`?